python/lsst/ts/electrometer/commander.py

```diff
@@ -30,40 +31,50 @@ class Commander:
         Whether the electrometer is connected or not.
     """
 
-    def __init__(self, log=None) -> None:
+    def __init__(self, log: None | logging.Logger = None) -> None:
         # Create a logger if none were passed during the instantiation of
         # the class
+        self.log: None | logging.Logger = None
         if log is None:
             self.log = logging.getLogger(type(self).__name__)
         else:
             self.log = log.getChild(type(self).__name__)
 
-        self.reader = None
-        self.writer = None
-        self.reply_terminator = b"\r"
-        self.command_terminator = "\r"
-        self.lock = asyncio.Lock()
-        self.host = tcpip.LOCAL_HOST
-        self.port = 9999
-        self.timeout = 2
-        self.connected = False
+        self.reader: None = None
+        self.writer: None = None
+        self.reply_terminator: bytes = b"\r"
+        self.command_terminator: str = "\r"
+        self.lock: asyncio.Lock = asyncio.Lock()
```

Comment:
Consider giving the lock a more descriptive name. Is this to lock I/O to the controller? If so, maybe use `self.controller_io_lock` or something on those lines.